keyboards/ergodox/readme.md

```diff
@@ -1,33 +1,38 @@
 # Getting started
 
-There are two main ways you could customize the ErgoDox EZ.
+There are two main ways you could customize the ErgoDox (EZ and Infinity)
 
 ## The Easy Way: Use an existing firmware file and just flash it
 
+This does not work for Infinity ErgoDox yet, you need to compile the firmware according to the instructions below
+
 1. Download and install the [Teensy Loader](https://www.pjrc.com/teensy/loader.html). Some Linux distributions already provide a binary (may be called `teensy-loader-cli`), so you may prefer to use this.
-2. Find a firmware file you like. You can find a few if these in the keymaps subdirectory right here. The file you need ends with .hex, and you can look at its .c counterpart (or its PNG image) to see what you'll be getting. You can also use the [Massdrop configurator](https://keyboard-configurator.massdrop.com/ext/ergodox) to create a firmware Hex file you like.
+2. Find a firmware file you like. You can find a few of these in the keymaps subdirectory right here. The file you need ends with .hex, and you can look at its .c counterpart (or its PNG image) to see what you'll be getting. You can also use the [Massdrop configurator](https://keyboard-configurator.massdrop.com/ext/ergodox) to create a firmware Hex file you like.
```

Comment:
I believe this section needs to be rewritten, since the hex files are no longer stored in the repository.